### PR TITLE
[11.0][MIG] account credit control dunning fees

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends on :
 - [x] https://github.com/OCA/account-financial-tools/pull/603

### CODE REVIEW

File: account_credit_control_dunning_fees/README.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+on the credit control level form.
+
+The amount will be used as fees values the currency will determine
+the currency of the fee. If the credit control line has not the
+same currency as the fees currency, fees will be converted to
+the credit control line currency.
+
+The product is used to compute taxes in reconciliation process.
+
+.. figure:: path/to/local/image.png
```
Comment (simahawk) [inline account_credit_control_dunning_fees/README.rst]: drop if not needed

File: account_credit_control_dunning_fees/view/policy_view.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <record id="add_dunning_fees_on_policy" model="ir.ui.view">
+    <field name="name">add dunning fees on policy</field>
+    <field name="model">credit.control.policy</field>
+    <field name="inherit_id" ref="account_credit_control.credit_control_policy_form" />
+    <field name="arch" type="xml">
+      <xpath expr="//notebook//notebook/page[//field[@name='email_template_id']]" position="after">
+        <page string="Fees">
+          <group>
```
Comment (simahawk) [inline account_credit_control_dunning_fees/view/policy_view.xml]: Do you need both groups? If yes, name at least the outer one.

**Review by simahawk — CHANGES_REQUESTED** [review]
LGTM, small things

**Comment by simahawk** [comment]
tests are failing because of missing dep from the other PR: add separated commit to drop later before merge w/ oca_depencies.txt pointing to that PR's branch.

**Review by simahawk — APPROVED** [review]

File: account_credit_control_dunning_fees/README.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Credit control dunning fees
+===========================
+
+This module extends the functionality of account_credit_control to add
+the notion of dunning fees on credit control lines.
+
+Installation
+============
+
+Just install it.
```
Comment (ghost) [inline account_credit_control_dunning_fees/README.rst]: drop this section?

**Review by ghost — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Comment by fclementic2c** [comment]
functional tests are Ok - good job !
:+1:

**Comment by simahawk** [comment]
@grindtildeath @yvaucher #603  merged, can we close this too?

**Comment by grindtildeath** [comment]
@simahawk For me, migration is done, and this PR can be merged.

**Pull request merged**